packages/ods/src/components/phone-number/src/constants/phone-number-locale.ts

```diff
@@ -0,0 +1,14 @@
+enum ODS_PHONE_NUMBER_LOCALE {
```

Comment:
'fr' and 'en' locales only?

Reply:
these are the translated country list we have available for now